CONTRIBUTING.md

```diff
@@ -0,0 +1,71 @@
+# Contributing
+
+## Code of Conduct
+
+This project and everyone participating in it is governed by a [Code of Conduct](code_of_conduct.md). By participating, you are expected to uphold this code. Please report unacceptable behavior to [quack@duckdb.org](mailto:quack@duckdb.org).
+
+## Eligibility
+
+Before submitting a contribution, please check whether your contribution is eligible.
+
+1. Before creating a new page, please do a search for the proposed page in the [existing documentation](https://duckdb.org/docs/).
+   You can start a search by using the search button or pressing <kbd>Ctrl</kbd> + <kbd>K</kbd> (Linux, Windows) or <kbd>Cmd</kbd> + <kbd>K</kbd> (MacOS).
+2. In general, guides for third-party tools using DuckDB should not be included in the DuckDB documentation. Rather, these tools and their documentation should be collected in the [`awesome-duckdb` community repository](https://github.com/davidgasquez/awesome-duckdb).
+
+## Adding a new page
+
+Thank you for contributing to the DuckDB documentation!
+
+Each new page requires at least 2 edits:
+* The creation of a new Markdown page with the documentation. Please follow the format of another `.md` file in the `docs` folder.
+* Add a link to the new page within `_data/menu_docs_dev.json`. This populates the dropdown menus.
+
+The addition of a new guide requires one additional edit:
+* Add a link to the new page within the Guides landing page: `docs/guides/index.md`
+
+**Please test your changes using the steps listed in the [Building](building.md) guide.**
+
+**When creating a PR, please check the box to "Allow edits from maintainers".**
+
+## Formatting
+
+### Syntax
+
+* Use [GitHub's Markdown syntax](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax) for formatting.
+* Please enclose code in blocks that are tagged with the appropriate language (e.g., \`\`\`sql CODE HERE \`\`\`). 
+* The title of the page should be encoded in the front matter's `title` property. The body of the pages should not repeat this title word-by-word.
+* Use **4 spaces** for indentation.
```

Comment:
I'd prefer we use the backtick syntax for code blocks, I'm not even sure if you can specify a syntax using the indentation version

Reply:
I mean indentation as in indentation in SQL code. Clarifying now.